src/PHPStan/Reflection/CallReflectionResolver.php

```diff
@@ -142,4 +150,38 @@ public function resolveParametersAcceptor($reflection, Node $node): ?ParametersA
 
         return $parametersAcceptor;
     }
+
+    /**
+     * @return FunctionReflection|MethodReflection|null
+     */
+    private function resolveConstantString(ConstantStringType $constantStringType, Scope $scope)
+    {
+        $value = $constantStringType->getValue();
+
+        // 'my_function'
+        $functionName = new Name($value);
+        if ($this->reflectionProvider->hasFunction($functionName, null)) {
+            return $this->reflectionProvider->getFunction($functionName, null);
+        }
+
+        // 'MyClass::myStaticFunction'
+        $matches = Strings::match(
+            $value,
+            '#^([a-zA-Z_\\x7f-\\xff\\\\][a-zA-Z0-9_\\x7f-\\xff\\\\]*)::([a-zA-Z_\\x7f-\\xff][a-zA-Z0-9_\\x7f-\\xff]*)\\z#'
```

Comment:
Could add regex101.com link and extract to constant? It would make it easier to test in case of bug

Reply:
Done.